protocol/migrations/sqlite/000001_init.up.db.sql

```diff
@@ -62,6 +62,7 @@ CREATE TABLE IF NOT EXISTS user_messages (
     command_state INT
 );
 
+CREATE INDEX idx_album_id on user_messages(local_chat_id, album_id);
```

Comment:
It's very common to add the table name to the index name, since it helps to understand which index is applied to which table.

But the more important question I have is, should we change an existing migration to add an index? Shouldn't we create a new migration file? cc: @cammellos

Reply:
Yes, we need a new migration file. The migrations that have already been applied are skipped, so this index would never apply to people just updating their version.

Reply:
Thanks @jrainville :)